shapely/geometry/geo.py

```diff
@@ -123,5 +168,19 @@ def asShape(context):
 
 
 def mapping(ob):
-    """Returns a GeoJSON-like mapping"""
+    """
+    Returns a GeoJSON-like mapping from an object which
+    implements __geo_interface__
```

Comment:
I would be explicit here that this includes shapely's own geometries (since I suppose that is probably the main usage of this function in practice), like ".. from a Geometry or any object which implements `__geo_interface__`"

Reply:
Great point, I added that clarification.